modules/cluster-logging-collector-log-forward-examples.adoc

```diff
@@ -0,0 +1,385 @@
+=== Additional RFC5424 syslog parameters
+
+The following parameters apply to RFC5424:
+
+* appName: The APP-NAME part of the syslog-msg header. Must be specified for `RFC5424`.
+* msgID: The MSGID part of the syslog-msg header. Must be specified for `RFC5424`.
+* procID: The PROCID part of the syslog-msg header. Must be specified for `RFC5424`.
+
+[id=cluster-logging-collector-log-forward-examples-secure]
+== Forwarding logs using secure mechanisms
+
+The following example forwards logs using multiple security methods:
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "LogForwarding"
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+   - name: elasticsearch-secure
+     type: "elasticsearch"
+     url: https://elasticsearch.sample.example.com:9200 <1>
+   - name: elasticsearch-secret
+     type: "elasticsearch"
+     url: https://elasticsearch.secret.example.com:9200
+     secret:
+        name: es-secret <2>
+   - name: elasticsearch-insecure
+     type: "elasticsearch"
+     url: elasticsearch.insecure.example.com.svc:9200
+  pipelines:
+   - name: infrastructure-logs
+     inputRefs:
+     - infrastructure
+     outputRefs:
+     - elasticsearch-secure
+   - name: application-logs
+     inputRefs:
+     - application
+     outputRefs:
+     - elasticsearch-secret
+   - name: audit-logs
+     inputRefs:
+     - audit
+     outputRefs:
+     - elasticsearch-insecure
+----
+<1> To use server-authenticated TLS, specify a secure URL, with the `https` prefix.
+<2> To use client-authenticated TLS, specify a secret with a secure URL.
+
+[id=cluster-logging-collector-log-forward-examples-ns]
+== Forwarding logs from specific projects
+
+The following example forwards application logs from specific projects:
+
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  pipelines:
+   - name: my-app <1>
+      inputRefs: <2>
+       - inputest
+      outputRefs: <3>
+       - default
+  inputs:
+   - name: inputest <4>
+      application:
+        namespaces: <5>
+        - project1
+        - project2
+----
+<1> Optional. Specify a name for the pipeline.
+<2> Specify the input to use.
+<3> Specify `default` to send the logs to the internal Elasticsearch instance.
+<4> Specify a name for the input.
+<5> Specify the projects from which to send logs.
+<6> Specify the output to use.
+
+[id=cluster-logging-collector-log-forward-examples-multi]
+== Forwarding logs to multiple outputs
+
+The following example forwards each log source to a different log aggregator using different protocols.
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1alpha1"
+kind: "LogForwarding"
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+   - name: elasticsearch-instance <1>
+     type: "elasticsearch"
+     url: elasticsearch.example.com:9200
```

Comment:
url: https://elasticsearch.example.com:9200

Reply:
Removed this topic.